paas-deploy.sh

```diff
@@ -0,0 +1,26 @@
+# Set CF_HOME in a temp dir so that jobs do not share or overwrite each others' credentials.
+export CF_HOME="$(mktemp -d)"
+trap 'rm -r $CF_HOME' EXIT
+
+cf api https://api.cloud.service.gov.uk
+
+# Note: the actual name of the environment variable is determined
+# by what you enter into the Credentials Binding Plugin
+cf auth "$CF_USER" "$CF_PASSWORD"
+
+cf target -o csd-sso -s sandbox
+
+echo '  env:' >> manifest.yml
+echo '      OIDC_CLIENT_ID: '$OIDC_CLIENT_ID >> manifest.yml
+echo '      OIDC_CLIENT_SECRET: '$OIDC_CLIENT_SECRET >> manifest.yml
+echo '      OIDC_CLIENT_ISSUER: '$OIDC_CLIENT_ISSUER >> manifest.yml
+echo '      SERVER_NAME: '$SERVER_NAME >> manifest.yml
```

Comment:
You can write lots of lines to a file using `EOL`, and I'd recommend `cat` rather than `echo`.

```bash
cat >manifest.yaml <<EOL
    OIDC_CLIENT_ID: ${OIDC_CLIENT_ID}
    OIDC_CLIENT_SECRET: ${OIDC_CLIENT_SECRET}
    OIDC_CLIENT_ISSUER: ${OIDC_CLIENT_ISSUER}
    SERVER_NAME: ${SERVER_NAME}
EOL
```